exporter/awsprometheusremotewriteexporter/auth.go

```diff
@@ -135,3 +147,25 @@ func cloneRequest(r *http.Request) *http.Request {
 	}
 	return r2
 }
+
+// FormatHeaderValues will add the name/version pair to the User-Agent request
+// header. If the extra parameters are provided they will be added as metadata to the
+// name/version pair resulting in the following format.
+// "name/version (extra0; extra1; ...)"
+// The user agent part will be concatenated with this current request's User-Agent string.
+func FormatHeaderValues(req *http.Request, name, version string, extra ...string) {
```

Comment:
We don't need to export this function.

formatHeaderValues

I'd actually call this addUserAgent because this is the function adding the user agent.